core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -248,6 +274,13 @@ def complete
     private
 
     def detect_frontend_to_install(bundler_context)
+      if options[:payment_method] != 'none'
+        say_status :warning, set_color(
+          "You selected a payment method that might require manual integration with `solidus_starter_frontend`",
```

Comment:
I think a better experience would be warning before the payment has been selected.